styles.css

```diff
@@ -0,0 +1,710 @@
+}
+
+.menu-bar {
+  float: left;
+  margin-left: 223px;
+  margin-top: 15px;
+  font-size: 25px;
+  color: #6070ff;
+}
+
+.hdiv {
+  display: flex;
+  flex-direction: row;
+}
+
+.head:hover {
+  color: red;
+}
+
+.menu-bar:hover {
+  color: red;
+}
+
+.head2 {
+  width: 259px;
+  height: 36px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: -50px;
+  margin-top: 0;
+}
+
+.p1 {
+  width: 55px;
+  height: 16px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.p2 {
+  width: 88px;
+  height: 16px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.p3 {
+  width: 29px;
+  height: 16px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.p4 {
+  width: 280px;
+  height: 220px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: -240px;
+  margin-left: 35px;
+}
+
+.link1 {
+  width: 29px;
+  height: 16px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.link2 {
+  width: 21px;
+  height: 16px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.link3 {
+  width: 65px;
+  height: 16px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.section2 {
+  width: 375px;
+  height: 718px;
+}
+
+.counter {
+  display: flex;
+  border-radius: 50%;
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 4px;
+}
+
+.up {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin-left: -40px;
+  margin-top: 20px;
+  width: 280px;
+}
+
+.back-link1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 30px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.back-link2 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 25px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.back-link3 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 65px;
+  height: 24px;
+  background: #ebebff;
```

Comment:
- [ ] According to the [DRY (DO NOT REPEAT YOURSELF) ](https://vanseodesign.com/css/dry-principles/), we should not repeat a block of code with the same functionality. But as you can see the CSS properties in `back-link1`, `back-link2` and `back-link3` are the same except for the width. So why don't you give your elements with classes `back-link1`, `back-link2` and `back-link3` one class and use this class to style all of them? :smile: Then when you are done you can specify the width for each.

  **PLEASE MAKE SURE YOU DO THIS TO MAKE YOUR CODE CLEAN** :pray: